R/ggplotMDS.R

```diff
@@ -309,8 +421,7 @@ ggplotMDS <- function(DGEdata,
                                             size = reflineSize)
         }
     }
-
-    return(list(plot = mdsplot, mdsobj = mds))
+    return(list(plot = mdsplot, mdsobj = mds.data))
```

Comment:
Simple call to **list(plot = mdsplot, mdsobj = mds.data)** will have the same affect

Reply:
@saranya-ag - what is the response here? Thank you